UKHomeOffice/cerberus-service
# COP-6478: Add Script to Post tasks into Camunda for testing

## Description
Add Cypress request to post tasks to Cerberus camunda before running tests. 
Fix / adapt tests for the latest changes.

## To Test
npm cypress:runner
run all the tests
observe the Cypress log, tasks will be posted to camunda & it would be available on Cerberus UI.

## Developer Checklist

\* Required
